modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/Region.java

```diff
@@ -9,13 +9,14 @@
 import jakarta.validation.constraints.NotBlank;
 import jakarta.validation.constraints.NotEmpty;
 import jakarta.validation.constraints.NotNull;
+import java.io.Serializable;
 import lombok.Data;
 
 /**
  * The regions of the Cloud Service Provider.
  */
 @Data
-public class Region {
+public class Region implements Serializable {
 
```

Comment:
Add serialVersionUID to avoid potential incompatibilities during deserialization.